src/components/ae-icon/ae-icon.vue

```diff
@@ -36,53 +43,74 @@ export default {
      */
     face: {
       type: String,
-      validator: value => ['round'].includes(value),
+      validator: value => ['', 'round'].includes(value),
+      default: '',
     },
 
     /**
-     * Set manually the size of the font icon
+     * Rotation of icon in degrees
      */
-    size: String,
+    rotate: {
+      type: [String, Number],
+      default: 0,
+    },
+  },
+  computed: {
+    style() {
+      return {
+        ...this.rotate && { transform: `rotate(${this.rotate}deg)` },
+      };
+    },
+    actualName() {
```

Comment:
Why is there a need for this function?

Reply:
It is to fix compatibility with previous names of AeIcon. Also, I think that `chevron` + rotation is actually better than `left-more`.